build.gradle

```diff
@@ -432,3 +439,9 @@ wrapper {
     println "Added checksum to wrapper properties"
   }
 }
+
+
+artifacts {
+  'default' buildDependenciesZip
+  'default' buildNoDependenciesZip
```

Comment:
For some reason only the `deps` file gets generated.  This is the same even if the order of these is swapped.

In the `elasticsearch` repo:

```
bash-3.2$ ./gradlew :x-pack:plugin:ml:assemble --include-build ../ml-cpp
.
.
.
> Task :x-pack:plugin:ml:extractNativeLicenses FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':x-pack:plugin:ml:extractNativeLicenses'.
> Cannot expand ZIP '/Users/dave/ml-cpp/build/distributions/ml-cpp-8.2.0-SNAPSHOT-nodeps.zip' as it does not exist.
```

And looking in the `ml-cpp` repo after that:

```
ls -l build/distributions
total 449712
-rw-rw-r--  1 dave  dev  44010914 Mar  3 11:09 ml-cpp-8.2.0-SNAPSHOT-deps.zip
-rw-rw-r--  1 dave  dev  44297817 Mar  3 11:09 ml-cpp-8.2.0-SNAPSHOT.zip
```